_code-samples/batch/js/multiAccountBatch.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * XRP Ledger Batch Transactions Tutorial
+ *
+ * This tutorial demonstrates how to use the Batch transaction feature (XLS-56)
+ * to perform a multi-account batch transaction.
+ * Concept doc: https://xrpl.org/docs/concepts/transactions/batch-transactions
+ * Reference doc: https://xrpl.org/docs/references/protocol/transactions/types/batch
+*/
+
+import xrpl from "xrpl"
+
+const client = new xrpl.Client("wss://s.devnet.rippletest.net:51233/")
+await client.connect()
+
+// Create and fund wallets
+console.log("=== Funding new wallets from faucet... ===")
+const { wallet: alice } = await client.fundWallet()
+const { wallet: bob } = await client.fundWallet()
+const { wallet: charlie } = await client.fundWallet()
+const { wallet: thirdPartyWallet } = await client.fundWallet()
```

Comment:
```suggestion
const [
  { wallet: alice },
  { wallet: bob },
  { wallet: charlie },
  { wallet: thirdPartyWallet }
] = await Promise.all([
  client.fundWallet(),
  client.fundWallet(),
  client.fundWallet(),
  client.fundWallet()
])
```
Just to speed up wallet generation to like 4ish seconds.